beeline/src/test/org/apache/hive/beeline/TestBeelineArgParsing.java

```diff
@@ -111,7 +111,7 @@ public void testSimpleArgs() throws Exception {
     String args[] = new String[] {"-u", "url", "-n", "name",
       "-p", "password", "-d", "driver", "-a", "authType"};
     org.junit.Assert.assertEquals(0, bl.initArgs(args));
-    Assert.assertTrue(bl.connectArgs.equals("url name password driver"));
+    Assert.assertTrue(bl.connectArgs.equals("url " + System.getenv().get("USER") +" password driver"));
```

Comment:
where is the test for sudo user?

Reply:
Can't set sudo user in System.getEnv its unmodifiable properties.